TF_Tooling_Web/src/TouchFreeToolingTypes.ts

```diff
@@ -51,15 +51,16 @@ export class TouchFreeInputAction {
 // Used to translate the raw actions that come across the websocket (<WebsocketInputActions>) and
 // convert them into the Tooling-friendly <TouchFreeInputAction> format.
 export function ConvertInputAction(_wsInput: WebsocketInputAction): TouchFreeInputAction {
-    const yPosition = window.innerHeight - _wsInput.CursorPosition.y;
+    const yPosition = (window.innerHeight - _wsInput.CursorPosition.y) / window.devicePixelRatio;
```

Comment:
This is doing the operations in the wrong order, so the cursor ends up offset as we subtract the unscaled window height

Reply:
I have updated this!